lib/src/main/java/space/taran/arklib/Lib.kt

```diff
@@ -20,11 +26,30 @@ data class LinkData(
     }
 }
 
+@Parcelize
 data class ResourceId(
     val dataSize: Long,
     val crc32: Long
-) {
+) : Parcelable {
     companion object {
+
+        class Converter {
+            @TypeConverter
+            fun fromString(str: String): ResourceId {
+                val idAry = str.split(":")
+                return ResourceId.create(
+                    idAry[0].toLong(),
+                    idAry[1].toLong()
+                )
+            }
+
+            @TypeConverter
+            fun toString(id: ResourceId) =
+                id.dataSize.toString()
+                    .plus(":")
```

Comment:
Let's make `':'` a constant and replace it's value with `'-'`.

Reply:
`-` because arklib is using this format, not really mandatory, but could make things easier when we'll move to arklib index

Reply:
Alright, then we should import the same constant in storages that are still in Navigator. For example scores, stats and tags. I don't, do we have a plan to moves these to arklib-android too?

Reply:
Yes, we need to move them as well. I just decided to keep things simple for the moment. We can add it to the next bounty.